routers/init.go

```diff
@@ -107,7 +107,7 @@ func InitWebInstallPage(ctx context.Context) {
 }
 
 // InitWebInstalled is for global installed configuration.
-func InitWebInstalled(ctx context.Context) {
+func InitWebInstalled(ctx context.Context, extraDBPrepare func() error) {
```

Comment:
```
// extraDBPrepare is an optional callback function, it's called immediately after the ORM system is ready.
// the main purpose is to fill the database with test fixtures during integration tests.
// Because in integration tests, the database should have the contents before other "init" calls, to make it behave like a production procedure.
// Ideally, the "extraDBPrepare" it could be refactored to a separate function to be called before "InitWebInstalled" to guarantee the database is ready ahead. 
```

Reply:
Update: as discussed, there could be other solutions to avoid `extraDBPrepare`

Reply:
reverted.